tests/zfs-tests/tests/functional/channel_program/lua_core/tst.stack_gsub.ksh

```diff
@@ -0,0 +1,33 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2020 by Delphix. All rights reserved.
+#
+
+. $STF_SUITE/tests/functional/channel_program/channel_common.kshlib
+
+#
+# DESCRIPTION:
+#       Overflowing the C stack using recursive gsub() should be handled
+#       gracefully.  gsub() uses more stack space than typical, so
+#	LUAI_MAXCCALLS must be set lower to ensure that we don't overflow
```

Comment:
I don't see where you're setting `LUAI_MAXCCALLS` lower so wondering if this comment is no longer valid.

Reply:
Thanks, this was based on an older revision.  Fixed.